cmd/psql/psql.go

```diff
@@ -0,0 +1,60 @@
+package psql
+
+import (
+	"errors"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/lakebase"
+	"github.com/spf13/cobra"
+)
+
+func New() *cobra.Command {
+	return newLakebaseConnectCommand()
+}
+
+func newLakebaseConnectCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "psql [DATABASE_INSTANCE_NAME] [-- PSQL_ARGS...]",
+		Short:   "Connect to the specified Database Instance",
+		Args:    cobra.MinimumNArgs(0),
+		GroupID: "database",
+		Long: `Connect to the specified Database Instance.
+
+This command requires a psql client to be installed on your machine for the connection to work.
```

Comment:
What is the difference between the database instance (arg[0] here) and the database name (via the -dbname flag in psql)?

The description can be a bit confusing; from the code it seems we connect to a database inside the database instance?